app/src/main/java/org/wikipedia/descriptions/DescriptionEditView.java

```diff
@@ -198,6 +199,7 @@ private void setDarkReviewScreen(boolean enabled) {
             cancelButton.setColorFilter(enabled ? whiteRes : ResourceUtil.getThemedColor(getContext(), R.attr.main_toolbar_icon_color), PorterDuff.Mode.SRC_IN);
             headerText.setTextColor(enabled ? whiteRes : ResourceUtil.getThemedColor(getContext(), R.attr.main_toolbar_title_color));
             ((DescriptionEditActivity) activity).updateStatusBarColor(enabled ? android.R.color.black : ResourceUtil.getThemedAttributeId(getContext(), R.attr.main_status_bar_color));
+            ((DescriptionEditActivity) activity).updateNavigationBarColor(enabled ? ContextCompat.getColor(activity, android.R.color.black) : ResourceUtil.getThemedAttributeId(getContext(), R.attr.paper_color));
```

Comment:
Is this right? One of these is a `ColorInt`, and the other is a `ColorRes`...

Reply:
Right, it looks not that good to me. Changed.